packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3581,6 +3583,7 @@ describe('plugin-meetings', () => {
           assert.calledWith(canUserResumeSpy, payload.info.userDisplayHints);
           assert.calledWith(canUserRaiseHandSpy, payload.info.userDisplayHints);
           assert.calledWith(canUserLowerAllHandsSpy, payload.info.userDisplayHints);
+          assert.calledWith(canUserLowerSomeoneElsesHandSpy, payload.info.userDisplayHints);
```

Comment:
nitpick: This in itself doesn't actually check that the return value of this is actually used anywhere - there should be a check in here either for a call to `inMeetingActions.set` (via the `inMeetingActionsSetSpy`, which seems to be unused in this set of tests), or the last argument to the call to `TriggerProxy.trigger` should be checked explicitly for content (though doing it this way you would need a test for the true and false values for any of the fields set on `inMeetingActions`).